packages/devtools_app/lib/src/screens/deep_link_validation/deep_link_list_view.dart

```diff
@@ -73,9 +79,12 @@ class _DeepLinkListViewState extends State<DeepLinkListView>
     );
   }
 
-  int _getReleaseVariantIndex(List<String> variants) {
+  int _getDefaultVariantIndex(
```

Comment:
this probably shouldn't be called variant anymore

Reply:
same for other places in the code that are shared between ios and android.

Reply:
renamed to configuration